README.md

```diff
@@ -244,11 +245,9 @@ tickets = client.tickets.include(:users)
 # Or client.tickets(:include => :users)
 # Does *NOT* make a request to the server since it is already loaded
 tickets.first.requester # => #<ZendeskAPI::User id=...>
-```
 
-OR
+# OR
```

Comment:
Can we do this `###` or `####` because `#` should be for title (I am not sure what the norms are at zendesk yet. 

```suggestion
### OR
```

Reply:
that's now inside a code block, I've done this change because it's more consistent with the other code blocks below.

Reply:
Very good point about the CI rules @tagpro 💯